torchmetrics/metric.py

```diff
@@ -221,6 +231,72 @@ def wrapped_func(*args, **kwargs):
 
         return wrapped_func
 
+    def sync(
+        self,
+        dist_sync_fn: Optional[Callable] = None,
+        process_group: Optional[Any] = None,
+        should_sync: bool = True,
```

Comment:
Do we need `op`?

Reply:
No need as `reduction_fn` is automatically inferred from `self._reductions`